src/main/resources/templates/fragments/user-footer.html

```diff
@@ -6,6 +6,7 @@
                     <li><a href="https://learn.civilservice.gov.uk/cookies">Cookies</a></li>
                     <li><a href="https://learn.civilservice.gov.uk/privacy">Privacy</a></li>
                     <li><a href="https://learn.civilservice.gov.uk/contact-us">Contact us</a></li>
+                    <li><a href="/accessibility/statement">Accessibility statement</a></li>
```

Comment:
If this is either an absolute URL like the ones above, or a dynamic reference to the LPG-UI then we don't need a duplicate accessibility statement page. Could you try replacing the tag with a templated version e.g. `<a th:href="${lpgUiUrl} + '/accessibility-statement'">` ?